### PR TITLE
Update header and T&C links on weekly page

### PR DESCRIPTION
**Pull request by lmath**
## Why are you doing this?
Keeping things up to date and tidy. 

[**Trello Card**](https://trello.com)

## Changes

* Update header image
* Remove headline from the product page hero
* GridImage -> Grid Picture
* Update the T&C link copy

## Screenshots
(compare to current state of the page here: https://support.theguardian.com/uk/subscribe/weekly)

| mobile        | tablet           | big wide web screen  |
| ------------- |-------------| -----|
|     |  |  |

### CODE REVIEW

**Comment by lmath** [comment]
The css in `weeklySubscriptionLanding.scss` is mostly copied from `showcase.scss` as the product page hero needs to look really similar.

It makes sense to me that there isn't a gridPicture.scss because grid pictures are used all over the place, but if there is going to be crossover for these header images sometimes, is there somewhere a common style for them should live in this case? (thanks to @akash1810 for suggesting I think about that, but alas, I'm still not sure how to go about it)

**Comment by walaura** [comment]
if we are committed to this type of layout it does make sense! we could wrap `ProductPageHero` to make a `ProductPageHeroWithCovers` that applies this type of treatment. My only worry is the cover images might all have slightly different positioning requirements to the point we won't be able to do much other than removing the left border?

File: assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx [hunk 1]
```diff
@@ -73,18 +73,35 @@ const content = (
       footer={<Footer />}
     >
       <ProductPagehero
-        headline="Become a Guardian Weekly subscriber"
```
Comment (walaura) [inline assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: this was the only instance of `headline` being used, worth checking with the designers (@blongden73 commented yesterday that he wanted to tweak its colour too) if this element will reappear again in any headers or we can remove it entirely

File: assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx [hunk 2]
```diff
@@ -135,11 +152,11 @@ const content = (
       </ProductPageContentBlock>
       <ProductPageContentBlock>
         <ProductPageTextBlock title="Promotion terms and conditions">
-          <p>Offer subject to availability. Guardian News and Media Limited (&ldquo;GNM&rdquo;) reserves the right to withdraw this promotion at any time. For full promotion terms and conditions visit <a target="_blank" rel="noopener noreferrer" href={`https://subscribe.theguardian.com/p/WWM99X/terms?country=${subsCountry}`}>subscribe.theguardian.com/p/WWM99X/terms</a>
+          <p>Offer subject to availability. Guardian News and Media Limited (&ldquo;GNM&rdquo;) reserves the right to withdraw this promotion at any time. For full 6 for 6 promotion terms and conditions, see <a target="_blank" rel="noopener noreferrer" href={`https://subscribe.theguardian.com/p/WWM99X/terms?country=${subsCountry}`}>here</a>.
```
Comment (walaura) [inline assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: yay! non-url links 😄

**Pull request merged**